CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+# [1.2.0 (June 30th, 2021)](https://github.com/eclipse/eclipse.jdt.ls/milestone/92?closed=1)
+ * enhancement - Add a preference to control the severity of not covered maven plugin execution. See [#1770](https://github.com/eclipse/eclipse.jdt.ls/pull/1770).
+ * enhancement - Allow folding `static` blocks. See [#1777](https://github.com/eclipse/eclipse.jdt.ls/issues/1777).
+ * enhancement - Add deprecated property to CompletionItem and SymbolInformation. See [#695](https://github.com/eclipse/eclipse.jdt.ls/issues/695).
+ * enhancement - deprecated is not populated in textDocument/documentSymbol. See [#1301](https://github.com/eclipse/eclipse.jdt.ls/issues/1301).
```

Comment:
1301 is a duplicate of 695, please remove it

Reply:
done in https://github.com/eclipse/eclipse.jdt.ls/commit/b6ea99e0318d3a0c417b4e9516726f05ad729ef9